grpc/python/BUILD

```diff
@@ -57,7 +57,7 @@ python_repackage(
 python_repackage(
     name = "protocol-cluster-pkg",
     src = ":protocol-cluster-src",
-    src_package = "protobuf.cluster",
+    src_package = "cluster",
```

Comment:
I'm not 100% sure if this will work with the current implementation of `python_repackage` (not sure if it replaces `import cluster` or just `cluster`) so we should test that ASAP